src/php/Command/Runtime/ConfigNormalizer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Command\Runtime;
+
+final class ConfigNormalizer
+{
+    /**
+     * @param array<string, list<string>> $result [ns => [path1, path2, ...]]
```

Comment:
Could you describe what this function is doing?

What type of `array` is `$configLoader`?

Reply:
I will do that 👍